src/dawg.cpp

```diff
@@ -17,7 +17,8 @@
 */
 #include "dawg.h"
 
-#include <boost/config.hpp>
+// #include <boost/config.hpp>
```

Comment:
Is there a reason to keep this commented out code around?

Reply:
Not at all, missed it when cleaning up. Will remove it.